openedx_webhooks/lib/jira/decorators.py

```diff
@@ -0,0 +1,29 @@
+# -*- coding: utf-8 -*-
+"""
+Decorators for working with JIRA.
+"""
+
+from __future__ import absolute_import, division, print_function, unicode_literals
+
+from functools import wraps
+
+from jira import JIRA
+
+
+def inject_jira(f):
+    """
+    Inject authenticated JIRA client (``jira``) if one is not supplied.
+    """
```

Comment:
This could use more explanation: the first argument to the function has to be the jira argument.